src/main/resources/io/jenkins/plugins/folderauth/FolderAuthorizationStrategyManagementLink/index.jelly

```diff
@@ -22,21 +22,32 @@
                     <input type="text" placeholder="${%filterPlaceholder}" class="filter" id="globalRoleFilter"/>
                     <div class="role-container" id="globalRoleContainer">
                         <j:forEach items="${it.globalRoles}" var="globalRole" indexVar="index">
-                            <div class="role" roleName="${globalRole.name}">
+                            <div class="role" roleName="${globalRole.name}" id="global-role-${globalRole.name}">
                                 ${%name}: ${globalRole.name}
                                 <br/>
                                 ${%sids}: ${globalRole.getSidsCommaSeparated()}
                                 <form action="${rootURL}/${it.urlName}/assignSidToGlobalRole" method="POST">
                                     <div class="form-row">
-                                        <label for="assign-sid-global-${index}" style="margin-right: 10px;">
+                                        <label for="assign-sid-global-${globalRole.name}" style="margin-right: 10px;">
                                             ${%assignSid}
                                         </label>
-                                        <input id="assign-sid-global-${index}" type="text" name="sid"/>
+                                        <input id="assign-sid-global-input-${globalRole.name}" type="text" name="sid"/>
                                         <input type="hidden" value="${globalRole.name}" name="roleName"/>
-                                        <input type="submit" style="margin-left: 10px;" value="${%submit}"
+                                        <input id="assign-sid-global-submit-${globalRole.name}" type="submit" style="margin-left: 10px;" value="${%submit}"
                                                class="submit"/>
                                     </div>
                                 </form>
+                                <form action="${rootUrl}/${it.urlName}/removeSidFromGlobalRole" method="POST">
```

Comment:
```suggestion
                                <form action="${rootUrl}/${it.urlName}/removeSidFromGlobalRole" method="POST">
                                <form action="${rootURL}/${it.urlName}/removeSidFromGlobalRole" method="POST">                                
```


`rootUrl` doesn't work. Should be `rootURL`. This mistake is repeated for folder and agent roles too.

![image](https://user-images.githubusercontent.com/11471599/68296808-76309080-00bb-11ea-9b28-f3d93a348155.png)

Reply:
sorry i've fixed this bug with the latest push